app-frontend/src/app/components/filterPane/filterPane.controller.js

```diff
@@ -86,29 +93,37 @@ export default class FilterPaneController {
     }
 
     initFilters() {
-        this.yearRange = {min: 2010, max: 2016};
-        let minAcquisitionDatetime =
-            (new Date(this.filters.minAcquisitionDatetime || NaN)).getFullYear() ||
-            this.yearRange.min;
-        let maxAcquisitionDatetime =
-            (new Date(this.filters.maxAcquisitionDatetime || NaN)).getFullYear() ||
-            this.yearRange.max;
-        this.yearFilters = {
-            minModel: minAcquisitionDatetime,
-            maxModel: maxAcquisitionDatetime,
-            options: {
-                floor: this.yearRange.min,
-                ceil: this.yearRange.max,
-                minRange: 1,
-                showTicks: 1,
-                showTicksValues: true,
-                pushRange: true,
-                draggableRange: true,
-                onEnd: this.onYearFiltersChange.bind(this)
+        this.cachedFilters = {};
+
+        let thisMonth = new Date().getMonth();
+        let thisYear = new Date().getFullYear();
+
+        // Changing from a 0-based to a 1-based indexing of months
+        let monthValues = [thisMonth + 1, thisMonth, thisMonth - 1].map(v => {
```

Comment:
I'm assuming the `v` is for `value` here but in my head I was expecting to see `m` or `mv`. Not a big deal but might be slightly clearer with a different variable name.